src/coreclr/src/utilcode/dacutil.cpp

```diff
@@ -107,7 +107,7 @@ LiveProcDataTarget::GetImageBase(
     // about right now is the base of mscorwks.
     //
 
-    if (wcscmp(name, MAIN_CLR_DLL_NAME_W))
+    if (wcscmp(name, TARGET_MAIN_CLR_DLL_NAME_W))
```

Comment:
Nit: This is the live instance - it does not need `TARGET_...`.

Reply:
Ok I can revert it.